src/Tribe/Main.php

```diff
@@ -596,4 +597,18 @@ public function embed_head() {
 		<link rel="stylesheet" id="tribe-tickets-embed-css" href="<?php echo esc_url( $css_path ); ?>" type="text/css" media="all">
 		<?php
 	}
+
+	/**
+	 * Enqueue and localize script for use in ticket delete alert dialogue
+	 *
+	 * @since 4.3
+	 */
+	public function add_ticket_deletion_alert() {
+		$url = $this->plugin_url . 'src/resources/js/ticket-delete-alert.js';
+		wp_enqueue_script( 'tribe_tickets_ticket_delete_alert', $url, array( 'jquery' ), self::VERSION, true );
```

Comment:
This all looks good, only comment is we should probably use `tribe_asset()` here (I think Gustavo mentioned this also, not sure if you discussed further).